starter-code/index.html

```diff
@@ -9,6 +9,42 @@
   <script type="text/javascript" src="./js/index.js"></script>
   <title>Ironhack cart</title>
 </head>
+
+
 <body>
+<div>
+  <div> 
+    <h1>Ironhack <br>Merchandising Shop</h1>
+  </div>  
+
+  <div class=itemInfo>
+    <span class=itemName>IronBubble-head</span>
+    <span class="price">$25</span>
```

Comment:
a good way to set this up in order to grab the inner html (the 25) is to set it up in a p tag then put $ then the span with the price. It would look something like this 
`
<p>
   $<span class="price">25</span>
</p>
`